stretch_ros.repos

```diff
@@ -51,3 +51,7 @@ repositories:
     type: git
     url: https://github.com/PickNikRobotics/rosparam_shortcuts
     version: ros2
+  realtime_tools:
+    type: git
+    url: https://github.com/ros-controls/realtime_tools
+    version: foxy-devel
```

Comment:
What are these used for?

Reply:
This's needed because it depends on control_msgs, and ros2_controllers depend on realtime_tools + control_msgs, and we changed the message definition for the trajectory action in control_msgs so we have to build it from source otherwise we'll have different control_msgs versions causing a seg-faults sometimes